server/passport-providers.js

```diff
@@ -159,5 +159,20 @@ module.exports = {
     scope: ['email'],
     link: true,
     failureFlash: true
+  },
+  'github-repo': {
```

Comment:
This will create two user identity objects connected to the same user and same github access. This is unneeded duplication and can cause problems later on. 

Keep looking for a way to increase user permission on the same user identity instead.